Dockerfile.base

```diff
@@ -1,24 +1,19 @@
-FROM ghcr.io/oracle/oraclelinux:7-slim
+FROM ghcr.io/oracle/oraclelinux:8-slim
```

Comment:
a suggestion - you can just use OL8 here since this image is not the final image . If you use OL8 you can re-use the yum commands (not repositories).  But OL8-slim works fine as well.